coalib/settings/ConfigurationGathering.py

```diff
@@ -240,9 +240,15 @@ def load_configuration(arg_list,
             Setting('config', re.escape(find_user_config(os.getcwd()))))
 
     targets = []
+    EXCLUDE_PREFIX = '!'
+    EXCLUDE_LENGTH = len(EXCLUDE_PREFIX)
+    exclude_sections = []
     # We don't want to store targets argument back to file, thus remove it
     for item in list(cli_sections['cli'].contents.pop('targets', '')):
-        targets.append(item.lower())
+        if item.startswith(EXCLUDE_PREFIX):
+            exclude_sections.append(item[EXCLUDE_LENGTH:].lower())
```

Comment:
maybe do
```
if item.startswith(EXCLUDE_PREFIX):
    item = item[len(EXCLUDE_PREFIX):]

targets.append(item.lower())
```
Then we have less code-copies^^

Reply:
I'm not really sure how this change works. Wouldn't it always append a section  to targets even if the section name is preceded by `!`

Reply:
yeah I have overseen that the lists appended to change, nvm ;)